### DISCUSSION TITLE
Expired SSL Certiface (Need Help pls.)

### DISCUSSION THREAD

**Discussion by HerrNorris** [Rapptz/discord.py#6042] [Q&A] [answered]
So, I'm having a problem with jour code. I think don't realy know where the problem is, but it seems to something with an expired SSL-Certificate. 

Can you help me with this please?

[ErrorMessage.txt](https://github.com/Rapptz/discord.py/files/5542618/ErrorMessage.txt)

Thanks in advance,
Herr Norris

**Comment by ShineyDev** [comment, marked as answer]
This is a duplicate of #4159.

**Reply by HerrNorris** [reply]
thx
